app/src/main/res/layout/city_item.xml

```diff
@@ -1,20 +1,95 @@
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools"
+<androidx.cardview.widget.CardView
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="100sp"
-    android:background="#0C1D46"
-    android:gravity="center_vertical"
-    android:layout_margin="16dp">
+    android:layout_height="wrap_content"
+    app:cardCornerRadius="16dp"
+    app:cardBackgroundColor="#0C1D46"
+    android:layout_margin="8dp"
+    app:cardUseCompatPadding="true"
+    app:cardElevation="4dp">
```

Comment:
I would add `android:elevation="4dp"` back. We want both of these field (the app:cardElevation and the android:elevation)